### ISSUE TITLE
FormatException on DSTU2

### ISSUE THREAD

**Issue by joshuaallenshaw** [ewoutkramer/fhir-net-api#819]
I am having a bit of trouble getting search results from a DSTU2 test server using the nuget packages for v1.0.0 .  The endpoint in question is https://pro171.open.allscripts.com/FHIR.  It requires OAuth2 against their authorization server so you can't run a blind test against it.

I am getting a bundle back, but the FHIR client is the throwing the FormatException.  The specific message is 

> The element 'p' uses the namespace 'http://www.w3.org/1999/xhtml', which is not allowed.

Here is the top of the body including the offending element.
`<Bundle xmlns="http://hl7.org/fhir"><type value="searchset" /><total value="18" /><entry><fullUrl value="https://pro171.open.allscripts.com/fhir/Patient/6015" /><resource><Patient><id value="6015" /><meta><profile value="http://hl7.org/fhir/StructureDefinition/daf-patient" /></meta><language value="en-US" /><text><status value="generated" /><p xmlns="http://www.w3.org/1999/xhtml">`

Is this an issue with my code, the server response, or the API?

The only part of my code that I can figure could be off is the client itself.   I'm using the defaults for the client.  Is there something I should change to make it accept the element?
`client = new FhirClient(endpoint);`

I don't have any control over the server, but I can certainly ask the maintainers if this xhtml is truly incorrect.  
As far as the API is concerned, I have tried this with the v1.1.0-beta1 packages too with no joy.

EDIT: If I set the preferred format to JSON, I get the data without error.  Not a fix so much as a work around.

**Comment by wmrutten** [comment]
The FHIR spec mandates the use of a Narrative.div element:
http://hl7.org/fhir/narrative.html#Narrative

The deserializer in the dotNET API is hardcoded accordingly, expecting a div tag with an xhtml namespace:
```
<text> 
    <status value="generated"/> 
    <div xmlns="http://www.w3.org/1999/xhtml">
        <!-- ... -->
    </div>
</text>
```
https://github.com/ewoutkramer/fhir-net-api/blob/0dcb52990fc9a500bae480124596fb86b30f829a/src/Hl7.Fhir.Core/Serialization/XmlDomFhirReader.cs#L119

I guess the issue doesn't occur for the JSON format, as JSON doesn't have namespaces.

**Comment by ewoutkramer** [comment]
The issue does not occur in Json because we forgot to look for it.  That's a known bug (https://github.com/ewoutkramer/fhir-net-api/issues/756) and has been fixed for the 1.2 release.

**Comment by ewoutkramer** [comment]
So, from 1.2 on, you will also get an error if you make this mistake in Json....
